hw/xbox/dsp/dsp_dma.c

```diff
@@ -137,9 +179,51 @@ static void dsp_dma_run(DSPDMAState *s)
             assert(false);
         }
 
+#ifdef DEBUG
+        char dsp_space_name = '?';
+        if (mem_space == DSP_SPACE_X) {
+            dsp_space_name = 'x';
+        } else if (mem_space == DSP_SPACE_Y) {
+            dsp_space_name = 'y';
+        } else if (mem_space == DSP_SPACE_P) {
+            dsp_space_name = 'p';
+        }
+#endif
+
+        DPRINTF("dsp dma block 0x%x (%s)\n"
+                "    next-block 0x%x%s\n"
+                "    control 0x%06x:\n"
+                "        dsp-interleave %d\n"
+                "        buffer-offset-writeback %d\n"
```

Comment:
nit: "buffer-offset-writeback" is only relevant for scratch buffers (`buf_id == 0xE || buf_id == 0xF`) or maybe also for some of the unknown buffer IDs.